libs/partners/cohere/langchain_cohere/rag_retrievers.py

```diff
@@ -63,12 +64,18 @@ class Config:
         """Allow arbitrary types."""
 
     def _get_relevant_documents(
-        self, query: str, *, run_manager: CallbackManagerForRetrieverRun, **kwargs: Any
+        self,
+        query: str,
+        *,
+        run_manager: CallbackManagerForRetrieverRun,
+        documents: Optional[List[Dict[str, str]]] = None,
```

Comment:
ooc why are we passing documents into the retriever?

Reply:
It's being passed in through `**kwargs` either way (example [here] at the bottom (https://python.langchain.com/docs/integrations/retrievers/cohere). I am pulling it out here to make a check for the connectors. The Cohere SDK throws an error when both a connector and documents are specified, so passing `None` for connectors if documents are specified.